### PR TITLE
display max input vars

### PR DESCRIPTION
**Pull request by brianteeman**
Since php 5.3.9 there is a php setting for the number of max_inut_vars with a default value of 1000

This pr just adds a test to display the value in com_admin&view=sysinfo

Knowing the value is quite important now especially on sites with a large number of menus, usergroups or modules as if the value for max_input_vars is not big enough then saving will fail silently or will be truncated.

(Hopefully I will also be able to provide a plugin to test the value and see if it is high enough for the users site and display a warning message)

### CODE REVIEW

**Comment by zero-24** [comment]
@brianteeman 

>(Hopefully I will also be able to provide a plugin to test the value and see if it is high enough for the users site and display a warning message)

postinstall message?
Should we check this also on the installation? (maybe only as "warning")

**Comment by brianteeman** [comment]
There are good security reasons for having the max_input_vars set at 1000
by default in PHP so you would not want to blindly increase it if there is
no need to do so.

The main case where it becomes an issue is if you have a total of more than
1000 modules, menu items and usergroups. So it is only something we should
be warning users about if they are close to the limit or have reached the
limit.

So no its not suitable for a pre-installation check or a postinstall message

On 17 July 2015 at 10:36, zero-24 <notifications@github.com> wrote:

> @brianteeman <https://github.com/brianteeman>
>
> (Hopefully I will also be able to provide a plugin to test the value and
> see if it is high enough for the users site and display a warning message)
>
> postinstall message?
> Should we check this also on the installation? (maybe only as "warning")
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/7453#issuecomment-122229369>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by brianteeman** [comment]
Plugin coming soon as well.

File: administrator/components/com_admin/views/sysinfo/tmpl/default_phpsettings.php [hunk 1]
```diff
@@ -157,6 +157,14 @@
 					<?php echo JHtml::_('phpsetting.set', $this->php_settings['iconv']); ?>
 				</td>
 			</tr>
+						<tr>
```
Comment (malukenho) [inline administrator/components/com_admin/views/sysinfo/tmpl/default_phpsettings.php]: fix align

**Comment by widmann-it** [comment]
Workend

**Comment by DavidTrenkle** [comment]
I have tested this item :white_check_mark: successfully on 965d632474e407de5e29b0d73036514aeeb38d8f<br /><br />works<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7453">issues.joomla.org/joomla-cms/7453</a>.</sub>

**Comment by designbengel** [comment]
I have tested this item :white_check_mark: successfully on 965d632474e407de5e29b0d73036514aeeb38d8f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7453">issues.joomla.org/joomla-cms/7453</a>.</sub>

**Comment by dgt41** [comment]
@brianteeman can you rebase this on latest staging, there is a conflict most probably form the file administrator/language/en-GB/en-GB.com_admin.ini

**Comment by brianteeman** [comment]
I could if I knew how to but I don't. Each time I try it fails

On 24 October 2015 at 15:49, Dimitris Grammatiko <notifications@github.com>
wrote:

> @brianteeman <https://github.com/brianteeman> can you rebase this on
> latest staging, there is a conflict most probably form the file
> administrator/language/en-GB/en-GB.com_admin.ini
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/7453#issuecomment-150823948>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by zero-24** [comment]
@brianteeman please try

```
git pull origin maxvar
git pull upstream staging
git commit -am 'fix merge conflicts'
git push origin maxvar
```

**Comment by brianteeman** [comment]
Strange that when it  comes to some peoples PR the committer is happy
resolve merge conflicts

I;ve moved on. Happy to just close it

On 24 October 2015 at 15:58, zero-24 <notifications@github.com> wrote:

> @brianteeman <https://github.com/brianteeman> please try
>
> git pull origin maxvar
> git pull upstream staging
> git commit -am 'fix merge conflicts'
> git push origin maxvar
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/7453#issuecomment-150824313>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by zero-24** [comment]
Sorry I don't understand what you mean? Did it work or not?

**Comment by brianteeman** [comment]
Closed

**Pull request closed without merging**